apm-lambda-extension/extension/process_events.go

```diff
@@ -26,6 +26,7 @@ import (
 func ProcessShutdown() {
 	log.Println("Received SHUTDOWN event")
 	log.Println("Exiting")
+	agentDataServer.Close()
```

Comment:
Question: No objections to it but was/is adding this `.Close()` call here just house keeping or did the lack of this have a negative side-effect on the extension?

Reply:
This was just some house-keeping. I thought we should properly close the server when we know the lambda env is shutting down.